units/host_allocator.go

```diff
@@ -198,6 +199,25 @@ func (j *hostAllocatorJob) Run(ctx context.Context) {
 	// host-spawning phase
 	//////////////////////
 
+	numIntentHosts, err := host.CountIntentHosts(ctx)
+	if err != nil {
+		grip.Error(message.WrapError(err, message.Fields{
+			"runner":   hostAllocatorJobName,
+			"instance": j.ID(),
+			"distro":   j.DistroID,
+			"message":  "failed to count intent hosts",
+		}))
```

Comment:
Is it intentional not to return here? Also, if err is nil, `message.WrapError` will return nil so we don't need to check if it's nil to do the grip call

Reply:
Yes, I don't want any issue with this call disrupting the flow of the job since it's not a hard requirement. Removed the error if statement.